hbase-server/src/main/java/org/apache/hadoop/hbase/master/region/MasterRegionFactory.java

```diff
@@ -80,6 +83,10 @@
   public static final byte[] PROC_FAMILY = Bytes.toBytes("proc");
 
   private static final TableDescriptor TABLE_DESC = TableDescriptorBuilder.newBuilder(TABLE_NAME)
+    .setColumnFamily(ColumnFamilyDescriptorBuilder.newBuilder(HConstants.CATALOG_FAMILY)
```

Comment:
Oh, we add it here. Any need of a migration of existing installs?

Reply:
No, it will be created automatically.